src/lib/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,90 @@
+import styled from "@emotion/styled";
+
+type AvatarSizes = "extra large" | "large" | "medium" | "small" | "tiny";
+type AvatarStatus = "online" | "offline";
+
+export interface AvatarProps {
+  name: string;
+  size: AvatarSizes;
+  image?: string;
+  status?: AvatarStatus;
+}
+
+const diameter: { [key in AvatarSizes]: string } = {
+  "extra large": "80px",
+  large: "64px",
+  medium: "48px",
+  small: "32px",
+  tiny: "16px",
+};
+
+const fontSize: { [key in AvatarSizes]: string } = {
+  "extra large": "1.75rem",
```

Comment:
Probably we can reuse values from our theme. See `src/lib/theme/themes/base.ts`.
If not, just leave it as it is.
![image](https://github.com/daniknewgarden/exo-ui/assets/38109617/db161e07-c090-43eb-a1e2-bb1d15dd2901)

Reply:
Yeah i tried that but they looked kind of wonky and inconsistant. It would work if we change the sizes of the avatar to match the step of the font change as i did in my calculations;
i basically just did:
80px -> 1.75rem
64/80*1.75 = 1.4 and so on
so that the font size stays proportionally consistant to the size of the avatar.